tendrl/commons/utils/etcd_utils.py

```diff
@@ -0,0 +1,73 @@
+import etcd
+
+'''
+   Read from etcd
+   input params:
+       _int : type  - >  maps.NamedDict
+              value - >  Ns._int
+       key  : type  - >  string
+              value - >  attr to be fetched
+
+   return param:
+       dict - > if read is successful
+       None - > if key is not found
+'''
+
+def read(_int,key):
```

Comment:
Also I think it might be better to call it quorum instead of flag so that if any more such flags are added the name is online with purpose..